extensions/sql-database-projects/src/test/projectTree.test.ts

```diff
@@ -116,4 +116,22 @@ describe.skip('Project Tree tests', function (): void {
 			'MyNestedFolder2',
 			'MyFile2.sql']);
 	});
+
+	it('Should be able to parse and include relative paths outside project folder', function (): void {
+		const root = os.platform() === 'win32' ? 'Z:\\Level1\\Level2\\' : '/';
```

Comment:
There's no reason for any logic to check for this, but I find it funny that we're escaping beyond FS root.